rust/agents/scraper/migration/src/m20220805_000004_create_table_message.rs

```diff
@@ -0,0 +1,125 @@
+use crate::l20220805_types::*;
+use crate::m20220805_000001_create_table_domain::Domain;
+use crate::m20220805_000003_create_table_transaction::Transaction;
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Message::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Message::Id)
+                            .big_integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(ColumnDef::new(Message::TimeCreated).timestamp().not_null())
+                    .col(ColumnDef::new(Message::Origin).unsigned().not_null())
+                    .col(ColumnDef::new(Message::Destination).unsigned().not_null())
+                    .col(ColumnDef::new(Message::LeafIndex).unsigned().not_null())
+                    .col(ColumnDef::new_with_type(Message::Sender, Address).not_null())
+                    .col(ColumnDef::new_with_type(Message::Recipient, Address).not_null())
+                    .col(ColumnDef::new(Message::MsgBody).binary())
```

Comment:
Does Hasura not support fetching binary as hex? I'd rather store it as binary since that is what it is unless we have a specific reason. If we don't have one now but run into it during testing it will be an easy fix.

Reply:
Unsure how Hasura will handle it but fine to try this for now and see how it goes